MapboxNavigation/NavigationMapView.swift

```diff
@@ -1433,42 +1425,24 @@ extension NavigationMapView {
      
      - parameter coordinates: Coordinates which represent buildings locations.
      - parameter extrudesBuildings: Switch which allows to highlight buildings in either 2D or 3D. Defaults to true.
+     
+     - returns: Bool indicating if number of buildings found equals number of coordinates supplied.
      */
-    public func highlightBuildings(at coordinates: [CLLocationCoordinate2D], in3D extrudesBuildings: Bool = true) {
-        guard coordinates != currentCoordinatesForBuildingsToHighlight else { return }
-        currentCoordinatesForBuildingsToHighlight = coordinates
-        buildingHighlightPollingTimer?.invalidate()
-        highlightedBuildingIdentifiers = Set()
-        findAndHighlightBuildings(at: coordinates, in3D: extrudesBuildings)
-        buildingHighlightPollingTimer = Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { [weak self] timer in
-            self?.findAndHighlightBuildings(at: coordinates, in3D: extrudesBuildings)
-        }
+    public func highlightBuildings(at coordinates: [CLLocationCoordinate2D], in3D extrudesBuildings: Bool = true) -> Bool {
```

Comment:
If the order of the coordinates is unimportant, the argument should be a set rather than an array.

Reply:
I couldn't store `CLLocationCoordinate2D` in a `Set`. Leaving this as-is.